content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,5 +6,255 @@ weight: 1
+This guide lets you quickly evaluate Istio ambient service mesh. These steps require you to have
+a {{< gloss >}}cluster{{< /gloss >}} running a
+[supported version](/docs/releases/supported-releases#support-status-of-istio-releases) of Kubernetes ({{< supported_kubernetes_versions >}}). You can use any supported platform, for
+example [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or
+others specified by the
+[platform-specific setup instructions](/docs/setup/platform-setup/).
+
+Follow these steps to get started with ambient:
+
+1. [Download and install](#download)
+1. [Deploy the sample application](#bookinfo)
+1. [Adding your application to ambient](#addtoambient)
+1. [Secure application access](#secure)
+1. [Control traffic](#control)
+1. [Uninstall](#uninstall)
+
+## Download and install {#download}
+
+1.  Download the [alpha version of Istio](url) with support for ambient mesh. 
+    If you don’t have a Kubernetes cluster, you can set up using 
+    locally (e.g. using kind as below) or deploy one in Google or AWS Cloud:
+
+    {{< text bash >}}
+    $ kind create cluster --config=- <<EOF
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    name: ambient
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    EOF
+    {{< /text >}}
+
+1.  The ambient profile is designed to help you get started with ambient mesh.
+    Install Istio with the ambient profile on your Kubernetes cluster, using
+    the istioctl downloaded above:
+
+    {{< text bash >}}
+    $ istioctl install --set profile=ambient
+    {{< /text >}}
+
+1.  After running the above command, you’ll get the following output that indicates
+    these four components are installed successfully!
+
+    {{< text plain >}}
+    ✔ Istio core installed
+    ✔ Istiod installed
+    ✔ CNI installed
+    ✔ Ingress gateways installed
+    ✔ Ztunnel installed
+    ✔ Installation complete
+    {{< /text >}}
+
+## Deploy the sample application {#bookinfo}
+
+You’ll use the sample [bookinfo application](/docs/examples/bookinfo/), which is part of
+your Istio download from previous steps. In ambient mode, you deploy applications to
+your Kubernetes cluster exactly the same way you would
+without Istio. This means you can have your applications running in your Kubernetes before
+you enable ambient mesh, and have them join the mesh without needing to restart or
+reconfigure your applications.
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+$ kubectl apply -f samples/sleep/sleep.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/linsun/sample-apps/main/sleep/notsleep.yaml
+{{< /text >}}
+
+Note: sleep and notsleep are two simple applications that can serve as curl clients.
+
+Connect productpage to the Istio ingress gateway so you can access the bookinfo
+app from outside of the cluster:
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
```

Comment:
Since this is new stuff, maybe we should document using k8s gateways, instead of Istio ones? https://istio.io/latest/docs/setup/additional-setup/getting-started/#ip

Reply:
Our thought is that our existing user will also read it, and it would be important to show them existing APIs also work on ambient to minimize the impact to move to ambient.  We should definitely cover new APIs in the dedicated routing section as well.

Reply:
Makes sense, but then it would probably be good to add tabs for both APIs, like all the other ingress docs. Seems less than ideal to show how to get started with cool new stuff but not using the latest of other cool stuff.

Reply:
That sounds good, we can add that in a follow up PR, in fact a couple of people said they wanted to help with this page :)